### DISCUSSION TITLE
Asset generation through CLI

### DISCUSSION THREAD

**Discussion by o15a3d4l11s2** [romannurik/AndroidAssetStudio#256] [Q&A] [answered]
Is it possible to generate assets through a CLI? If not, I think this would be a good idea.

At the moment I am generating white-labels and it would be really nice to automate asset generation through this tool.

**Comment by romannurik** [comment, marked as answer]
This isn't currently possible, but you're welcome to modify the source code to support this. The simplest way to do this is probably with a tool like [Puppeteer](https://pptr.dev/)
